### ISSUE TITLE
can not order bare metal with terraform

### ISSUE THREAD

**Issue by hippo1024** [IBM-Cloud/terraform-provider-ibm#472]
Hello,

I have a Issue, Plz help this issue , thank you

### Terraform Version
version : terraform-provider-ibm_v0.14.0

### Affected Resource(s)
Please list the resources as a list, for example:
IBM Baremetal Instance(monthly)
failed server order

### Terraform Configuration Files
[Baremetal Order.txt](https://github.com/IBM-Cloud/terraform-provider-ibm/files/2748691/Baremetal.Order.txt)

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.
plz wait

### Expected Behavior
What should have happened?
not order Server

### Actual Behavior
What actually happened?
occur log : Windows Server 2012 R2 Standard Edition (64 bit) (2-4 Cores), price ID# 222581, has a Cores capacity restriction that does not match the capacity of Dual Intel Xeon Gold 6140 (36 Cores,
2.30 GHz), price ID# 202203. Please submit prices that are compatible. (HTTP 500)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
1. IBM Baremetal Window licenser has changed recently.
2. Recently IBM portal has been changed a lot and integration portal has been opened.
Integrated Portal: https://cloud.ibm.com/

Windows license details
My suspicion is a change in the Windows license.
Existing License: Windows Server 2012 R2 Standard Edition (64 bit) (2 Processor) - [$ 34.00]
Current license: Windows Server 2012 R2 Standard Edition (64 bit) (36 Core) [$ 162.00]

The two above seem to be related.

### References
I'dont know

**Comment by hkantare** [comment]
We will look into issue and get back..

**Comment by allmightyspiff** [comment]
I help support the softlayer api, and just wanted to chime in.

I'm not sure if this will be helpful since it is in python, but this is how the SLCLI looks up priceIds for items that have a capacity restriction.

https://github.com/softlayer/softlayer-python/blob/master/SoftLayer/managers/ordering.py#L374

```
"'https://api.softlayer.com/rest/v3.1/SoftLayer_Product_Package/837/getObject.json?objectMask=mask[id,keyName,activeSoftwareItems[id,keyName,prices[id,capacityRestrictionMaximum,capacityRestrictionMinimum]]]'

{
    "id": 837,
    "keyName": "DUAL_INTEL_XEON_PROCESSOR_SCALABLE_FAMILY_4_DRIVES",
    "activeSoftwareItems": [
        {
            "id": 4548,
            "keyName": "OS_WINDOWS_2012_R2_FULL_DC_64_BIT_2",
            "prices": [
                {
                    "capacityRestrictionMaximum": "4",
                    "capacityRestrictionMinimum": "2",
                    "id": 222677
                },
                {
                    "capacityRestrictionMaximum": "16",
                    "capacityRestrictionMinimum": "6",
                    "id": 222679
                },
                {
                    "capacityRestrictionMaximum": "20",
                    "capacityRestrictionMinimum": "20",
                    "id": 222681
                },
                {
                    "capacityRestrictionMaximum": "24",
                    "capacityRestrictionMinimum": "24",
                    "id": 222683
                },
                {
                    "capacityRestrictionMaximum": "28",
                    "capacityRestrictionMinimum": "28",
                    "id": 222685
                },
                {
                    "capacityRestrictionMaximum": "32",
                    "capacityRestrictionMinimum": "32",
                    "id": 222687
                },
                {
                    "capacityRestrictionMaximum": "36",
                    "capacityRestrictionMinimum": "36",
                    "id": 222689
                },
                {
                    "capacityRestrictionMaximum": "40",
                    "capacityRestrictionMinimum": "40",
                    "id": 222691
                },
                {
                    "capacityRestrictionMaximum": "48",
                    "capacityRestrictionMinimum": "48",
                    "id": 222693
                },
                {
                    "capacityRestrictionMaximum": "60",
                    "capacityRestrictionMinimum": "60",
                    "id": 222695
                },
                {
                    "capacityRestrictionMaximum": "64",
                    "capacityRestrictionMinimum": "64",
                    "id": 222697
                },
                {
                    "capacityRestrictionMaximum": "96",
                    "capacityRestrictionMinimum": "96",
                    "id": 222699
                }
            ]
```

**Comment by hkantare** [comment]
yes we identified the same ...We will fix the issue soon

**Comment by hkantare** [comment]
Fixed the issue and testing in in progress..
https://github.com/hkantare/terraform-provider-ibm/tree/fix/baremetal

**Comment by hippo1024** [comment]
@hkantare 
Is this version of Terraform a fixed version of the issue?

Did you find out what caused the problem?

Can I provide detailed information?

**Comment by hkantare** [comment]
We released v0.1.4.1 driver with a fix https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.14.1
We tested the scenario and its working as expected. We continue our testing to verify other scenarios mean while you can use the above driver.

**Comment by hkantare** [comment]
closing the issue
